webpack.config.js

```diff
@@ -12,6 +13,13 @@ const es6modulePaths = es6modules.map(module => {
   return path.join(__dirname, 'node_modules', module);
 });
 
+// If L10N env variable is set, we read all the locale directories and use
+// whatever we have there. This is done to make the l10n branch work with staging
+// locales, so localizers can see the result of their translations immediately.
+const availableStagingLocales = process.env.L10N
+  ? JSON.stringify(fs.readdirSync('./locales'))
```

Comment:
I hope we don't need to be smarter (for example filter our names starting with `.`, and/or keeping only directories).

but for now we can keep it simple.